solr/contrib/ltr/src/test/org/apache/solr/ltr/store/rest/TestModelManager.java

```diff
@@ -136,28 +129,46 @@ public void testRestManagerEndpoints() throws Exception {
     assert (qryResult.contains("\"name\":\"testmodel3\"")
         && qryResult.contains("\"name\":\"testmodel4\"") && qryResult
           .contains("\"name\":\"testmodel5\""));
-    /*
-     * assertJQ(LTRParams.MSTORE_END_POINT, "/models/[0]/name=='testmodel3'");
-     * assertJQ(LTRParams.MSTORE_END_POINT, "/models/[1]/name=='testmodel4'");
-     * assertJQ(LTRParams.MSTORE_END_POINT, "/models/[2]/name=='testmodel5'");
-     */
+
+    assertJQ(ManagedModelStore.REST_END_POINT + "/_DEFAULT_",
```

Comment:
There isn't a _DEFAULT_ model.  Not sure why this is doing anything. the /model-store endpoint should return all models, and /model-store/myModelName should return the contents of myModelName